LiteFarmOrg/LiteFarm
# Empty graph: error handling

To Test,

1. go to the sensor readings page
2. check the text when there are no readings available.

Screenshot: 

<img width="950" alt="Screen Shot 2022-12-22 at 3 43 50 PM" src="https://user-images.githubusercontent.com/20675885/209243790-3280db20-a5ff-4c3f-b2d4-61ac156d564c.png">
